pom.xml

```diff
@@ -343,8 +343,8 @@
                         </oldVersion>
                         <parameter>
                             <onlyModified>true</onlyModified>
-                            <breakBuildOnBinaryIncompatibleModifications>true</breakBuildOnBinaryIncompatibleModifications>
-                            <breakBuildBasedOnSemanticVersioning>true</breakBuildBasedOnSemanticVersioning>
+                            <breakBuildOnBinaryIncompatibleModifications>false</breakBuildOnBinaryIncompatibleModifications>
```

Comment:
Create a followup JIRA item to revert this after the major release.

Reply:
Internal Ref: [OKTA-379452](https://oktainc.atlassian.net/browse/OKTA-379452)